pyramid_ogcapi/__init__.py

```diff
@@ -66,6 +66,14 @@ def {route_name.lower()}(self, pyramid_request: pyramid.request.Request, request
     return None
 
 
+def path2route_name_prefix(path: str, route_prefix: Optional[str] = None) -> str:
+    return ("" if route_prefix is None else route_prefix) + (
```

Comment:
What would you think of changing the default route_prefix to ""? To me it would improve a bit the readability:
```suggestion
def path2route_name_prefix(path: str, route_prefix: Optional[str] = "") -> str:
    return route_prefix + (
```

Reply:
+1, Thanks :-)